browser/about_flags.cc

```diff
@@ -228,6 +228,11 @@ constexpr char kBraveSyncDescription[] =
 constexpr char kBraveVPNName[] = "Enable experimental Brave VPN";
 constexpr char kBraveVPNDescription[] = "Experimental native VPN support";
 
+constexpr char kBraveVPNDnsProtectionName[] = "Enable DoH for Brave VPN";
+constexpr char kBraveVPNDnsProtectionDescription[] =
+    "Automatically enable, if possible, DoH with Brave VPN on windows to avoid "
```

Comment:
I think "if possible" should be removed here. I think it just confuses things, but I don't think we're accurately describing it either. Maybe something like "Override DoH settings with Cloudflare dns if necessary to avoid leaking requests due to Smart Multi-Home Named Resolution"

Reply:
wdyt @pes10k ?

Reply:
yea, i agree, "if possible" makes it sound like we might silently fail, when (IIUC) its more "we might not need to in the first place"

Reply:
changed to @bridiver text